packages/runtime/container-runtime/src/gc/garbageCollection.ts

```diff
@@ -580,18 +592,16 @@ export class GarbageCollector implements IGarbageCollector {
 		);
 
 		// 4. Run the Sweep phase.
-		// It will delete sweep ready nodes and return a list of deleted node ids.
-		const deletedNodeIds = this.runSweepPhase(gcResult, sweepReadyNodeIds);
+		// It will initiate the deletion of sweep ready nodes by sending a sweep op. Once the op is ack'd, these nodes
+		// will be deleted.
```

Comment:
nit

```suggestion
		// It will initiate the deletion of sweep ready nodes by sending a sweep op. Each client - including this one -
		// will delete these nodes when it processes this op after it's sequenced.
```